HRIS.Services.Tests/Services/DashboardServiceUnitTest.cs

```diff
@@ -51,7 +51,7 @@ public async Task CalculateChurnRateTest()
 
         var result = await _dashboardService.CalculateEmployeeChurnRate();
         Assert.NotNull(result);
-        Assert.Equal("July", result.Month);
+        Assert.Equal("August", result.Month);
```

Comment:
This needs to be improved, next month it becomes the same issue

Reply:
The hot fix solves this problem, just resolve conflicts and update branch